tests/test_ws.py

```
@@ -0,0 +1,69 @@
+        websocket.send_json({"invalid": "data"})
+
+        response = websocket.receive_json()
+        assert response["type"] == "error"
+
+
+def test_websocket_unauthorized():
+    client = TestClient(app)
+    # No api_key or token
+    with pytest.raises(
+        Exception
+    ):  # TestClient raises when connection is closed with error code
```

Ruff (0.14.14)

[warning] Do not assert blind exception: `Exception`

(B017)